src/server/pkg/deploy/assets/assets.go

```diff
@@ -1106,6 +1167,12 @@ func WriteAssets(w io.Writer, opts *AssetOpts, objectStoreBackend backend,
 
 	encoder.Encode(ServiceAccount())
 	fmt.Fprintf(w, "\n")
+	if !opts.NoRBAC {
```

Comment:
I know this is superficial, but would `opts.RBAC` be clearer than `!opts.NoRBAC`?

Reply:
It definitely would be clearer, but it would flip the default. My thinking with this was that it's better to turn it on by default since there theoretically shouldn't be any reason not to have it outside of legacy support which should tail off over time.